modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -511,9 +518,21 @@ public void testPostCustomObjectEntryWithInvalidNestedSystemObjectEntries()
 	public void testPostCustomObjectEntryWithNestedSystemObjectEntry()
 		throws Exception {
 
+		_objectFieldLocalService.addCustomObjectField(
```

Comment:
Could we move this functionality to one of the TestUtils we are using?

I think it is going to be useful for testing system objects